### ISSUE TITLE
Passing url query parameters using nghttp 2?

### ISSUE THREAD

**Issue by msgharpu** [nghttp2/nghttp2#1006]
Is there a way to submit a get request with url parameters using nghttp? E.g is a query in following format possible?

https://testserver.com/auth?user=ABC&token=xyz

**Comment by tatsuhiro-t** [comment]
Yes, it should work.  Don't forget to quote the URI if you enter it in terminal.

```
$ nghttp "https://testserver.com/auth?user=ABC&token=xyz"
```

**Comment by msgharpu** [comment]
@tatsuhiro-t  I'm looking for a way using nghttp2 library in source code...
nghttp2_submit_request, takes care of headers, but in that how can I pass the url params?

**Comment by tatsuhiro-t** [comment]
Pass it in nghttp2_nv with the name ":path".
https://nghttp2.org/documentation/nghttp2_submit_request.html

**Comment by dio** [comment]
So `:path` by definition may contain query string?
